src/Nav.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import './App.css';
+import { Link } from 'react-router-dom'
```

Comment:
This is what you need to check out ESLint + Prettier for: There is some inconsistent formatting in the codebase.